QuizApp/Screens/HomePage/HomePageViewController.swift

```diff
@@ -0,0 +1,207 @@
+                equalTo: headerLabel.bottomAnchor,
+                constant: Constants.HomePageConstants.gpaViewTopPadding
+            ),
+            gpaView.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor,
+                constant: Constants.HomePageConstants.horizontalPadding
+            ),
+            gpaView.trailingAnchor.constraint(
+                equalTo: view.trailingAnchor,
+                constant: -Constants.HomePageConstants.horizontalPadding
+            )
+        ])
+    }
+
+    private func setupSubjectsLabelConstraints() {
+        NSLayoutConstraint.activate([
+            subjectsLabel.topAnchor.constraint(
+                equalTo: gpaView.bottomAnchor,
+                constant: Constants.HomePageConstants.subjectsLabelTopPadding
+            ),
+            subjectsLabel.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor,
+                constant: Constants.HomePageConstants.horizontalPadding
+            )
+        ])
+    }
+
+    private func setupTableViewConstraints() {
+        NSLayoutConstraint.activate([
+            tableView.topAnchor.constraint(
+                equalTo: subjectsLabel.bottomAnchor,
+                constant: Constants.HomePageConstants.tableViewTopPadding
+            ),
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            tableView.bottomAnchor.constraint(equalTo: bottomSeparatorLine.topAnchor)
+        ])
+    }
+
+    private func setupBottomSeparatorLineConstraints() {
+        NSLayoutConstraint.activate([
+            bottomSeparatorLine.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            bottomSeparatorLine.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            bottomSeparatorLine.heightAnchor.constraint(
+                equalToConstant: Constants.HomePageConstants.bottomSeparatorLine
+            ),
+            bottomSeparatorLine.bottomAnchor.constraint(
+                equalTo: view.bottomAnchor,
+                constant: Constants.HomePageConstants.bottomSeparatorLinePadding
+            )
+        ])
+    }
+
+    private func setupLogOutButtonConstraints() {
+        NSLayoutConstraint.activate([
+            logOutButton.leadingAnchor.constraint(
+                equalTo: view.leadingAnchor,
+                constant: Constants.HomePageConstants.logOutButtonLeadingPadding
+            ),
+            logOutButton.bottomAnchor.constraint(
+                equalTo: view.bottomAnchor,
+                constant: Constants.HomePageConstants.logoutButtonBottomPadding
+            ),
+            logOutButton.widthAnchor.constraint(equalToConstant: Constants.HomePageConstants.logoutButtonWidth),
+            logOutButton.heightAnchor.constraint(equalToConstant: Constants.HomePageConstants.logoutButtonHeight)
+        ])
+    }
+    
+    private func setupTableView() {
```

Comment:
You can have all of this in in tableView property itself and you won't need to have a separate method for applying delegate/data source and registering the cell. Just a tip do whatever.